### DISCUSSION TITLE
Creating .NET 9 MAUI Blazor Hybrid App

### DISCUSSION THREAD

**Discussion by halilix** [dotnet/maui#21045] [Q&A]
Hi, I have tried to create new MAUI Blazor Hybrid Application, but there is not even an option to choose .NET 9.0, in the wizard dialog, i have also tried to upgrade a 8.0 project to 9.0, didn't work aswell, also trying to update the nuggets to the preview release didn't work, can someone please give me complete introduction of creating MAUI Blazor Hybrid Application 9.0 app

**Comment by ghost** [comment]
@halilix I have tried Visual Studio Insiders and the option .NET 9.0 and .NET 10.0 preview is there. Visual Studio 2022 shows .NET 8.0 and .NET 9.0. Please update your Visual Studio to latest version and try again.
